torch/profiler/profiler.py

```diff
@@ -45,6 +45,33 @@ def _default_schedule_fn(_: int) -> ProfilerAction:
     """
     return ProfilerAction.RECORD
 
+def tensorboard_trace_handler(dir_name: str, worker_name: Optional[str] = None):
+    """
+    Outputs tracing files to directory of ``dir_name``, then that directory can be
+    directly delivered to tensorboard as logdir.
+    ``worker_name`` is used to distinguish different workers in distributed scenario,
```

Comment:
emphasize that `worker_name` need to be unique in distributed training scenario.